### DISCUSSION TITLE
Windows x64 Updated build for downloader plugin (Fix ERROR 1:71)

### DISCUSSION THREAD

**Discussion by spoonh** [pear-devs/pear-desktop#2726] [Show and tell]
Hey guys, was having issues downloading mp3s which is a feature of one of the plugins. I read through the build options and i got it to work before a published working fixed was published, so i wanted to share :) this app is awesome, and the devs who put a bunch of hard work towards this are amazing! lmk if it works for you, im new to program builds. 

[Download](https://mega.nz/file/WmoSzYQb#NENQRrH-vUI1FkeSdCFzkWcWwv4jEK57E_c3_KlDDqA)

**Comment by spoonh** [comment]
Also none of the code was actually written by me obviously, i just followed what was committed but hadnt been published to a build yet.

**Comment by IphoneCake** [comment]
Thank you for the work! I will test it soon!

**Comment by IphoneCake** [comment]
@spoonh Thanks for the work. It works perfectly !!

**Reply by spoonh** [reply]
> @spoonh Thanks for the work. It works perfectly !! 

Glad to hear, again the devs and whoever pushed the code to master is to thank i just built the exe lol. Have a good holiday :)

**Reply by b3hr3nss** [reply]
I ended up doing the same to get downloads working thanks for posting. @spoonh

**Comment by IphoneCake** [comment]
@spoonh I wish you the same. I think it’s great that you found a solution and were able to support the developer. The YouTube Music app is absolutely awesome.

**Comment by AymanHatime** [comment]
Hey, I had the ERROR 1:71 so I tried you soluce but I have a new error message when I try to download : "The server responded with a non 2xx status code in 40ce64......"

**Reply by Nakxin** [reply]
Same here

**Reply by IphoneCake** [reply]
@spoonh Once again, a download error occurs, and the same issue persists with the new version 3.7.0.0. Do you have a solution? Greetings!"
